recipe/install_clang_symlinks.sh

```diff
@@ -5,3 +5,10 @@ maj_version="${PKG_VERSION%%.*}"
 ln -s "${PREFIX}/bin/clang-${maj_version}" "${PREFIX}/bin/clang-cl"
 ln -s "${PREFIX}/bin/clang-${maj_version}" "${PREFIX}/bin/clang-cpp"
 ln -s "${PREFIX}/bin/clang-${maj_version}" "${PREFIX}/bin/clang"
+
+if [[ "$target_platform" == "linux-"* ]]; then
+  source ${RECIPE_DIR}/get_cpu_triplet.sh
+  CHOST=$(get_triplet $target_platform)
+  ln -s "${PREFIX}/bin/clang-${maj_version}" "${PREFIX}/bin/${CHOST}-clang"
+  echo "--sysroot ${PREFIX}/${CHOST}/sysroot" >> ${PREFIX}/bin/${CHOST}-clang.cfg
```

Comment:
Nah, that would require keeping them in sync

Reply:
Not that hard to avoid that... Fixed in https://github.com/conda-forge/clangdev-feedstock/pull/252/commits/ab4c0418b803f23a4314675a0de9e0a369fa126a for the forward-port to 17.x